README.md

```diff
@@ -107,5 +109,31 @@ This library gives you a Front controller (shown in the example) that can handle
 
 In the example above, routing is handled with Express's built-in `app[VERB]` methods, and the three parameters are set properly using express's built-in `:param` syntax. If you're looking for something more robust, you might be interested in [Express Simple Router](https://github.com/ethanresnick/express-simple-router). For authentication, check out [Express Simple Firewall](https://github.com/ethanresnick/express-simple-firewall).
 
+## Handling Errors
+
+This library provides an error contructor and a handler that you can use to return JSON API-compliant errors to the user. For example, here is how you would handle 404s.
+
+```javascript
+var API = require("json-api");
+var APIError = API.types.Error;
+
+// Your route definitions would go here, but if none match...
+
+app.use(function(req, res, next) {
```

Comment:
Rather than putting this example here, can you link to the line with the same code in the example repo? That way, we don't have to keep this code up to date in two places, and (more importantly) people won't get confused about what `Front` is.